lib/controllers/aliens.js

```diff
@@ -0,0 +1,33 @@
+const { Router } = require('express');
+const Alien = require('../models/Alien');
+
+module.exports = Router()
+  .post('/', async (req, res) => {
+    const alien = await Alien.insert(req.body);
```

Comment:
We should try to add `try/catch` blocks to all of our routes to ensure that errors don't go un-handled and break the API